scheduler/client/README.md

```diff
@@ -17,7 +17,7 @@ The setup of these is defined by server.Defaults(), which sets up the ice.MagicB
 * Thrift interface
 * Configuration Schema
 
-The actual implementations in scoot/scootapi/server handle Cloud Server API request handling from the Thrift interface down. The main elements here are:
```

Comment:
have a comment explaining what is offered in the client directory:
- ? main() exposing a cli that issues commands ?via thrift? to the scheduler
- ?a library of functions that issue commands ?via thrift? to the scheduler.  These functions can be used in a custom client.